### ISSUE TITLE
Navigation with Alt arrows skips over elements when moving away from grace notes and similar elements.

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#27671]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

This is likely to affect bends and some other similar elements - see closed issue [19926.](https://github.com/musescore/MuseScore/issues/19926)

When navigating with Alt + arrow keys it is now possible to move to the grace note.  However, all other elements such as articulation, lyrics, etc are skipped over when moving from the grace note or its main note to the next or previous notes.  Arrowing from the next or previous note towards the grace note does reveal all the elements in between.  But if someone is using Alt arrow to move through a score to pick up all elements they will never know some of these are there.

STEPS:
1.  Add 3 notes in a row to a score and place a variety of articulation, dynamics and some lyrics on each.
2. Add a grace note to the second note.
3. Move focus to the first note and use Alt + right arrow to move along the bar. Notice that the grace note is read and then it skips to the 3rd note missing the second note and everything attached to it.
4. Now move backwards through the bar from the 3rd note using Alt left arrow.  Notice that this time all the elements between 3rd and 2nd note are read, but after the grace note it skips straight to note 1 missing reading all the elements attached to note 1.

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5.1

### Regression

I was unable to check

### Operating system

 Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by zacjansheski** [comment]
Duplicate of https://github.com/musescore/MuseScore/issues/24160
